components/script/dom/subtlecrypto/ecdh_operation.rs

```diff
@@ -1004,53 +1146,61 @@ pub(crate) fn export_key(format: KeyFormat, key: &CryptoKey) -> Result<ExportedK
                 // 6.2.1.2 of JSON Web Algorithms [JWA].
                 // Step 3.3.3. Set the y attribute of jwk according to the definition in Section
                 // 6.2.1.3 of JSON Web Algorithms [JWA].
+                let public_key_err =
+                    Error::Operation(Some("Failed to export public key".to_string()));
+                let private_key_err =
+                    Error::Operation(Some("Failed to export private key".to_string()));
```

Comment:
It might be worth making these functions so that you don't always allocate here.


```suggestion
                let public_key_err = ||
                    Error::Operation(Some("Failed to export public key".to_string()));
                let private_key_err = ||
                    Error::Operation(Some("Failed to export private key".to_string()));
```